### PR TITLE
STORM-596: remove config topology.receiver.buffer.size

### PR DESCRIPTION
**Pull request by caofangkun**
topology.receiver.buffer.size has no effect and not necessary for now.
Should we just remove it ?

### CODE REVIEW

**Comment by caofangkun** [comment]
@harshach 
Could you please have a review on this PR? Thanks a lot.

**Comment by revans2** [comment]
This is technically a non-backwards compatible change, but I think it is OK.  I am +1.

File: storm-core/src/clj/backtype/storm/daemon/worker.clj [hunk 1]
```diff
@@ -355,8 +355,7 @@
     (:storm-id worker)
     (:receiver-thread-count worker)
     (:port worker)
-    (:transfer-local-fn worker)
-    (-> worker :storm-conf (get TOPOLOGY-RECEIVER-BUFFER-SIZE))
+    (:transfer-local-fn worker))
```
Comment (revans2) [inline storm-core/src/clj/backtype/storm/daemon/worker.clj]: There is an extra ')' here

**Comment by revans2** [comment]
No one else has decided to review this, and it still compiles and I am still +1 so I will merge it in.

**Pull request merged**